tests/subworkflows/nf-scil/preproc_dwi/main.nf

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env nextflow
```

Comment:
Don't bother with this file. Remove the whole test directory for the subworkflow in fact. We'll do `nf-test` directly.